file/lib/file_source_endpoint.ex

```diff
@@ -29,21 +32,40 @@ defmodule Membrane.RTC.Engine.Support.FileSourceEndpoint do
               ],
               ssrc: [
                 spec: RTP.ssrc_t(),
+                default: 1,
```

Comment:
I'm wondering whether we should provide a default implementation of [generating a random SSRC](https://www.rfc-editor.org/rfc/rfc3550#section-8) (or if we can simply leave it up to the user)